src/main/java/com/google/sps/servlets/MealServlet.java

```diff
@@ -57,16 +94,17 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         // Get meal
         // GET meal/
         if (pathInfo == null || pathInfo.equals("/")) {
-            searchMeal(request, response);
+            getMealList(request, response);
             return;
         }
 
-        
         if (pathInfo.split("/").length == 2) {
             String idString = pathInfo.replaceAll("/", "");
```

Comment:
I know I named it like that myself first time, but now seems like name doesn't match variable anymore.
Maybe `requestType` or `requestInfo` will be better.